README.md

```diff
@@ -100,6 +100,10 @@ if let Ok(event) = MenuEvent::receiver().try_recv() {
 }
 ```
 
+## Documentation
```

Comment:
Actually, could we replace this with a badge from https://shields.io/badges/docs-rs at the top?